pkgs/desktops/deepin/deepin-anything/default.nix

```diff
@@ -12,6 +12,13 @@ mkDerivation rec {
     sha256 = "1kvyffrii4b012f6ld1ih14qrn7gg5cxbdpbkac0wxb22hnz0azm";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://git.archlinux.org/svntogit/community.git/plain/trunk/0001-linux-5.6.patch?h=packages/deepin-anything-arch";
+      sha256 = "1pgi1cbbxpi6c9fywi8nd0q6xr9v5zxnrvzqkhcbz34iv509vs7l";
+    })
```

Comment:
```suggestion
      name = "0001-linux-5.6.patch";
      url = "https://git.archlinux.org/svntogit/community.git/plain/trunk/0001-linux-5.6.patch?h=packages/deepin-anything-arch";
      sha256 = "1pgi1cbbxpi6c9fywi8nd0q6xr9v5zxnrvzqkhcbz34iv509vs7l";
    })
```

Reply:
Changed to use a patch directly from a PR in the upstream repository.